backend/metamodel/syson-sysml-metamodel/src/main/java/org/eclipse/syson/sysml/impl/ElementImpl.java

```diff
@@ -530,44 +530,42 @@ else if (eNotificationRequired())
 
     /**
      * <!-- begin-user-doc -->
+     * The full ownership-qualified name of this Element, represented in a form that is valid according to the KerML
+     * textual concrete syntax for qualified names (including use of unrestricted name notation and escaped characters, as
+     * necessary). The qualifiedName is null if this Element has no owningNamespace or if there is not a complete
+     * ownership chain of named Namespaces from a root Namespace to this Element.
      * <!-- end-user-doc -->
      * @generated NOT
      */
     @Override
     public String getQualifiedName() {
-        boolean qualifiedNameContainsNull = false;
+        String selfName = this.getName();
```

Comment:
It seems you add the same modifications in this file and in your other PR https://github.com/eclipse-syson/syson/pull/299/. Please remove these modifications, they don't need to be in two different PRs.
Please check every other files in this PR.